LoopKit/DeviceManager/CGMManager.swift

```diff
@@ -20,15 +20,40 @@ public enum CGMReadingResult {
     case error(Error)
 }
 
-public struct CGMManagerStatus {
+public struct CGMManagerStatus: Equatable {
     // Return false if no sensor active, or in a state where no future data is expected without user intervention
     public var hasValidSensorSession: Bool
 
     public var lastCommunicationDate: Date?
+
+    public var device: HKDevice?
```

Comment:
How does this differ from the `device` attribute on the CGMManager itself?  What would it mean if this returned a different device than the one on cgmmanager? Which one should the hosting app use?

Reply:
It is the same. I don't know why, but the `PumpManager` returns the device as part of the `PumpManagerStatus`, but the `CGMManager` returns the device on its own. This just makes it uniform (so all "status" from the manager goes through the `*Status` object). I'm happy to remove the device on the `CGMManager` itself, but I wanted to make as few changes as possible.

Reply:
Ok, I don't think we should have two different ways of doing this on CGMManager. As for which one is better, I don't have a strong opinion. Having it on status means that changes to device can be 'events', so clients don't have to poll for changes, if they are concerned with that.  But it also means additional costs in terms of including infrequently changed data on a frequently passed struct.

Reply:
Updated. Removed `device` on `CGMManager` protocol. Also updated `SupportInfoProvider` protocol to use `CGMManagerStatus`.